x/group/msgs.go

```diff
@@ -44,18 +42,14 @@ func (m MsgCreateGroup) ValidateBasic() error {
 		return sdkerrors.Wrap(err, "admin")
 	}
 
-	return m.validateMembers()
-}
-
-func (m MsgCreateGroup) validateMembers() error {
-
-	err := validateMembers(m.Members)
-	if err != nil {
-		return err
-	}
-	return nil
+	return strictValidateMembers(m.Members)
 }
 
+// ValidateBasic performs stateless validation on a group member, such as
+// making sure the address is well-formed, and the weight is non-negative.
+// Note: in state, a member's weight MUST be positive. However, in some Msgs,
+// it's possible to set a zero member weight, for example in
+// MsgUpdateGroupMembers to denote that we're removing a member.
```

Comment:
I replaced some `math.NewNonNegativeDecFromString` by `math.NewPositiveDecFromString`, e.g. everywhere where it's stored in state, according to this comment. Just want to double check this logic is correct.

Reply:
makes sense